FakeSign failure - binary already signed

http://dotnet-ci.cloudapp.net/job/roslyn-internal_master_win_vsi_p1/309

00:14:42.895   PE file is already strong-name signed.
00:14:42.895   Use the -f (force) option to sign d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\Binaries\Obj\SyntaxVisualizerControl\Release\Roslyn.SyntaxVisualizer.Control.dll anyway.
00:14:42.907 d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\build\Targets\VSL.Imports.targets(288,5): error MSB3073: The command ""C:\Users\dotnet-bot\.nuget\packages\FakeSign\0.9.2\tools\FakeSign.exe" "d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\Binaries\Obj\SyntaxVisualizerControl\Release\Roslyn.SyntaxVisualizer.Control.dll"" exited with code 1. [d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\src\Tools\Source\SyntaxVisualizer\SyntaxVisualizerControl\SyntaxVisualizerControl.csproj]

Also,

00:14:42.907   PE file is already strong-name signed.
00:14:42.909   Use the -f (force) option to sign d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\Binaries\Obj\SyntaxVisualizerDgmlHelper\Release\Roslyn.SyntaxVisualizer.DgmlHelper.dll anyway.
00:14:42.916 d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\build\Targets\VSL.Imports.targets(288,5): error MSB3073: The command ""C:\Users\dotnet-bot\.nuget\packages\FakeSign\0.9.2\tools\FakeSign.exe" "d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\Binaries\Obj\SyntaxVisualizerDgmlHelper\Release\Roslyn.SyntaxVisualizer.DgmlHelper.dll"" exited with code 1. [d:\j\workspace\roslyn-internal_master_win_vsi_p1\Open\src\Tools\Source\SyntaxVisualizer\SyntaxVisualizerDgmlHelper\SyntaxVisualizerDgmlHelper.vbproj]

--- a/build/Targets/VSL.Imports.targets
+++ b/build/Targets/VSL.Imports.targets
@@ -283,9 +283,9 @@
                      ('$(Language)' == 'C#' OR '$(Language)' == 'VB') AND
                      '$(BuildingProject)' == 'true'"
           Inputs="@(IntermediateAssembly)"
-          Outputs="@(IntermediateAssembly);&quot;$(PostCompileBinaryModificationSentinelFile)&quot;">
+          Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
 
-    <Exec Command="&quot;$(FakeSignToolPath)&quot; -f &quot;@(IntermediateAssembly)&quot;" />
+    <Exec Command="&quot;$(FakeSignToolPath)&quot; &quot;@(IntermediateAssembly)&quot;" />
 
   </Target>
 

